helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -121,6 +121,11 @@ splunkPlatform:
       enabled: false
       storagePath: "/var/addon/splunk/exporter_queue"
 
+  # Option to disable or enable fsync for filestorage extension used by the agent. Enabling this option will ensure
+  # database integrity at the cost of performance.
+  # Refer to: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/extension/storage/filestorage#file-storage
+  fsyncEnabled: false
```

Comment:
Can we abide by the default from upstream?

Reply:
Yeah we could, see the comment above